src/libraries/Microsoft.Extensions.Configuration.CommandLine/src/Microsoft.Extensions.Configuration.CommandLine.csproj

```diff
@@ -1,13 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
   </PropertyGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
```

Comment:
just for curiosity, why we are adding the new dependencies here ?

Reply:
The library wasn't building for .NETCoreApp before correctly because of the `TargetFramework` being set to `$(DefaultNetCoreTargetFramework)` which doesn't exist in this repo. Probably a leftover from the dotnet/extensions migration. Fixing this requires the added references.